docs: update clustermesh troubleshooting guide (#26798)
cilium-team/cilium: Documentation/operations/troubleshooting_clustermesh.rst

           └  remote configuration: expected=true, retrieved=true, cluster-id=3, kvstoremesh=false, sync-canaries=true
           └  synchronization status: nodes=true, endpoints=true, identities=true, services=true

 #. Validate that the required TLS secrets are set up properly. By default, the
    following TLS secrets must be available in the namespace in which Cilium is
    installed:

    * ``clustermesh-apiserver-server-cert``, which is used by the etcd container
      in the clustermesh-apiserver deployment. Not applicable if an external etcd
      cluster is used.

    * ``clustermesh-apiserver-admin-cert``, which is used by the apiserver/kvstoremesh
      containers in the clustermesh-apiserver deployment, to authenticate against the
      sidecar etcd instance. Not applicable if an external etcd cluster is used.

    * ``clustermesh-apiserver-remote-cert``, which is used by Cilium agents, and
      optionally the kvstoremesh container in the clustermesh-apiserver deployment,
      to authenticate against remote etcd instances (either internal or external).

>     If any of above secrets is not configured correctly, there will be potential error message like below::

Reviewer: The words 'above' and 'below' give translators some trouble, can we change to 'prior' and 'following'?
```suggestion
    If any of the prior secrets is not configured correctly, there will be a potential error message like the following::
```

    If the configuration is not found, check the following:

    * The ``cilium-clustermesh`` Kubernetes secret is present and correctly
      mounted by the Cilium agent pods.

    * The secret contains a file for each remote cluster with the filename matching
      the name of the remote cluster (as provided by the ``--cluster-name`` argument

>       or the ``cluster-name`` ConfigMap option).

Reviewer: I think we can remove the parentheticals here.

```suggestion
      the name of the remote cluster as provided by the ``--cluster-name`` argument
      or the ``cluster-name`` ConfigMap option.
```

    * Each file named after a remote cluster contains a valid etcd configuration
      consisting of the endpoints to reach the remote etcd cluster, and the path
      of the certificate and private key to authenticate against that etcd cluster.

>       Additional files may represent the certificate and private key themselves.

Reviewer: This sentence is a little unclear, I'm not sure what it means. I tried to add some context:

```suggestion
      Additional files may be included in the secret to provide the certificate and private key themselves.
```

    * The ``/var/lib/cilium/clustermesh`` directory inside any of the Cilium agent
      pods contains the files mounted from the ``cilium-clustermesh`` secret.
      You can use
      ``kubectl exec -ti -n kube-system ds/cilium -c cilium-agent -- ls /var/lib/cilium/clustermesh``

>       to list the files therein present.

Reviewer: I think `present` and `therein` are redundant. Can you pick one and remove the other?
